.github/scripts/package_push.sh

```diff
@@ -0,0 +1,58 @@
+# Deal with Debian Package First
```

Comment:
Add `#!/bin/bash` to all scripts. Also ensure your version variables are valid (not null/empty) before tagging and pushing.

Reply:
Done. If version is not set, script will echo that the push failed and then exit the script.